render.js

```diff
@@ -0,0 +1,21 @@
+const Renderer = function () {
+    const renderPosts = function (posts) {
+        $("#posts").empty()
+        for (let post of posts) {
+            let newPostId = post.id
+            $("#posts").append("<div class='pcon' data-id="+ `${newPostId}` + "> <h1>" + post.text + "</h1></div>")
```

Comment:
save each of these as variables. When you put something so big inside the `.append()` method it makes the code difficult to follow. The best code is the most simple and most clean. Adding more veriables can help. 

`const newPostId = post.id
const postBox = $("<div class='pcon' data-id="+ `${newPostId}` + "> <h1>" + post.text + "</h1></div>")
$("#posts").append(postBox)`